layouts/partials/home.html

```diff
@@ -5,7 +5,7 @@ <h2>{{ .Site.Params.info }}</h2>
     {{ with .Site.Params.social }}
     <ul>
       {{ range sort .}}
-      <li><a href="{{ .url }}">{{ .name }}</a></li>
+      <li><a href="{{ .url }}"><i class="fab {{ .icon }}">&nbsp;{{ .name }}</i></a></li>
```

Comment:
Not all of the Font Awesome v5 starts with `fab` some of them are `fas`, `fal`, `far` as in `solid`, `light`, `brands`. I would suggest to add another attribute in social config such as `type = b`, `type = r`, etc.